tests/testthat/error/test-subscript-loc.txt

```diff
@@ -174,6 +174,11 @@ i It must be numeric or character.
 Error: Must extract element with a single valid subscript.
 x Lossy cast from `foo` <double> to <integer>.
 
+> with_tibble_rows(vec_as_location2(TRUE))
+Error: Must remove row with a single valid subscript.
+x The subscript `foo(bar)` has the wrong type `logical`.
```

Comment:
`foo(bar)`?

Reply:
This is what `subscript_arg` is set to in the `with_tibble_rows()` tests.